helper/helper.py

```diff
@@ -3,13 +3,13 @@
 from sqlalchemy import MetaData, create_engine
 from sqlalchemy.orm import sessionmaker
 
-DB_USER = 'root'
+
 DB_PASSWORD = 'root' # noqa
-DB_URL = 'localhost:5432/perfectfit'
 
 
-def get_db_session():
-    db_loc = 'postgresql://' + DB_USER + ":" + DB_PASSWORD + "@" + DB_URL
+def get_db_session(db_host = 'localhost', db_user = 'root'):
+    db_url = db_host + ":5432/perfectfit"
```

Comment:
Maybe the port (5432) could be combined with db_host? That way users can change that too in case it's not on the default port.

Reply:
Good point.